pre-commit-config.yaml

```
@@ -0,0 +1,11 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)